### PR TITLE
Clean up src

### PR DESCRIPTION
**Pull request by Luni-4**
- Remove unused files
- Change confusing names

### CODE REVIEW

**Comment by Luni-4** [comment]
@calixteman 

Can we merge this one?

File: rust-code-analysis-cli/src/main.rs [hunk 1]
```diff
@@ -78,7 +78,7 @@ fn act_on_file(language: Option<LANG>, path: PathBuf, cfg: &Config) -> std::io::
         };
         action::<Dump>(&language, source, &path, pr, cfg)
     } else if cfg.metrics {
-        let cfg = MetricsCfg {
+        let cfg = SpacesCfg {
```
Comment (calixteman) [inline rust-code-analysis-cli/src/main.rs:81]: I don't understand why you changed the name here, maybe a `sed s//g` issue

Comment (Luni-4) [reply rust-code-analysis-cli/src/main.rs:81]: Because I renamed `MetricsCfg` into `SpacesCfg` since it's a configuration for functions spaces.

**Comment by calixteman** [comment]
Thx

**Pull request merged**